### ISSUE TITLE
Led firmware control only white channel

### ISSUE THREAD

**Issue by Sennevds** [ptvoinfo/zigbee-configurable-firmware#277]
I've bought the premium features add created following config:
![image](https://github.com/ptvoinfo/zigbee-configurable-firmware/assets/3531609/c526c6b6-5ab5-44ea-91ce-294b9efaa4ee)
It's for a 24v led strip which has 4 separate inputs (RGBW) This works and I can perfectly control the color and brightness.
I can't find how to set the PWM to 0 on the RGB channels and only control the white channel.
Is this possible?

**Comment by ptvoinfo** [comment]
@Sennevds 
Did you configure your firmware in FirmwareConfigLED.exe?

**Comment by Sennevds** [comment]
Yes, I've found out that if I send #ffffff all color leds go out but for some reason the white leds won't turn on I will check my pcb on errors this evening.
Still strange that only when sending #ffffff it will turn on the white leds I would think that there would be a toggle in Home Assistant between rgb and white

**Comment by ptvoinfo** [comment]
@Sennevds It is by design. The white channel is active only when colors are inactive. So, you can color or white modes.

**Comment by Sennevds** [comment]
Okay thanks for the info, probably just a weird way Home Assistant works with rgbw lights.
Because I need to manually set the color to white it's not possible to easily toggle between rgb and white.

**Comment by Sennevds** [comment]
@ptvoinfo I've created a test setup (with the none led firmware because other device and not having a premium for that one).
The PWM signal is working great for 3 pin's (P16, P20 and P17) but it's not working on pin P23.
This is my config:
![image](https://github.com/ptvoinfo/zigbee-configurable-firmware/assets/3531609/baf7d048-a8f7-46e3-a1f4-0ac6514ce000)
[pool.txt](https://github.com/ptvoinfo/zigbee-configurable-firmware/files/15119520/pool.txt)

**Comment by ptvoinfo** [comment]
As I see, P23 is used for an external oscillator. Or did you assemble a custom board without it?

**Comment by ptvoinfo** [comment]
I cannot open this page (product not found). The most boards have 32 kHz oscillators. Check your device.

**Comment by Sennevds** [comment]
Checked pinout and was indeed connected to oscillator. 
Last thing about the rgbw configuration. The light is recognized as a RGB light and you automatically switch to the white LEDs when you select white color. Home assistant has a option to identify a light as rgbw and then has a different control for the white LEDs. Is this possible?
